homeassistant/components/tado/config_flow.py

```diff
@@ -66,7 +67,9 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    async def async_step_user(self, user_input=None):
+    async def async_step_user(
```

Comment:
Can we type `validate_input()` too so whole file is completely typed?

Reply:
Done and done. :)